benchmark/hdf5/benchmark_knowhere.h

```diff
@@ -27,6 +27,9 @@
 #include "knowhere/index/index_factory.h"
 #include "knowhere/version.h"
 
+static const size_t default_build_thread_num = 8;
```

Comment:
https://en.cppreference.com/w/cpp/thread/thread/hardware_concurrency ? Please don't forget to check for 0 if you decide to go this path.

Reply:
Hi Alex, this variable is only used in benchmark